language/tools/move-package/tests/test_sources/external_dev_dep/Move.progress

```diff
@@ -1,11 +1,11 @@
-RESOLVING DEPENDENCIES IN A FROM Root WITH ../resolvers/successful.sh
-../resolvers/successful.sh stderr:
+[1;32mRESOLVING DEPENDENCIES IN[0m A [1;32mFROM[0m Root [1;32mWITH[0m ../resolvers/successful.sh
```

Comment:
Also agreed on this one -- not sure what has caused it to change its output (it is in a sense legitimate, because the escape codes do exist to colour things green, but I don't know why your change has prompted them to start appearing in tests).